src/plugins/safeSnap/utils/index.ts

```diff
@@ -171,3 +173,83 @@ export async function fetchTextSignatures(
   const { results } = await response.json();
   return results.map(signature => signature.text_signature);
 }
+
+export const formatVotesResolutions = (votes: Vote[]) => {
+  let forVotes = new ethers.utils.BigNumber(0);
+  let againstVotes = new ethers.utils.BigNumber(0);
+  let abstainVotes = new ethers.utils.BigNumber(0);
+
+  const formattedVotes = votes.map(vote => {
+    const { vp, vp_by_strategy, choice } = vote;
+    if (choice === 1) {
+      forVotes = forVotes.add(ethers.utils.parseEther(String(vote.vp)));
```

Comment:
We could probably save ethers.utils.parseEther(String(vote.vp)) as a variable for performance.